config/locales/client.en.yml

```diff
@@ -2752,6 +2752,11 @@ en:
       filtered_replies_hint:
         one: "View this post and its reply"
         other: "View this post and its %{count} replies"
+      filtered_replies_viewing:
+          one: "Viewing %{count} reply to"
```

Comment:
I wonder if we need the "to" when there's just one?

Reply:
I think so, the user's avatar is added after the "to": 

<img width="775" alt="image" src="https://user-images.githubusercontent.com/368961/102236585-adc65a80-3ec1-11eb-960f-0d3a1c11a458.png">

Reply:
That makes sense!